extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js

```diff
@@ -771,6 +771,8 @@ export class AmpLightboxViewer extends AMP.BaseElement {
         anim.add(0.9, tr.setStyles(transLayer, {
           opacity: tr.numeric(1, 0.01),
         }), 0.1, EXIT_CURVE_);
+      } else {
```

Comment:
In QA Round 3 we decided that it was weird for the black background to fade in first. Originally, setting carousel opacity to 0 was implemented to avoid the transition image moving towards an image that already existed. If that transition is not happening, then we shouldn't set the carousel opacity to zero.